backups/backup.py

```diff
@@ -13,30 +12,41 @@
 log = logging.getLogger("Importer")
 
 
+capella_version: str = os.getenv("CAPELLA_VERSION", "")
+
+
 def run_importer_script() -> None:
     log.debug("Import model from TeamForCapella server...")
-    subprocess.run(
+
+    connection_type: str = get_connection_type()
+
+    command: list[str] = ["/opt/capella/capella"]
+
+    if is_capella_5_x_x():
+        command.append("--launcher.suppressErrors")
```

Comment:
Can you test if this is properly applied. When the run is successful, there should be no "JVM terminated" log at the end of the subprocess run.